### DISCUSSION TITLE
A way to play the pronunciation of the original text.

### DISCUSSION THREAD

**Discussion by HubKing** [FilipePS/Traduzir-paginas-web#378] [Ideas]
It currently has one pronunciation button, and it plays the pronunciation of the translated text. But I need to listen to the pronunciation of the original text. Imagine that you were learning French by reading some French web document. You saw a new French word and used this extension to see its meaning. You would want to know how the French word sounds, not how the English translation sounds.

![image](https://user-images.githubusercontent.com/10900489/156762221-6b8b7283-d14b-4e29-b543-b3850b5198ef.png)

A similar extension has pronunciation buttons for both the original and the translated text. So, instead of having the pronunciation button on the bottom, how about adding the button to each of the text above like so:

![image](https://user-images.githubusercontent.com/10900489/156763013-bdacbb8e-1b2a-413c-ba60-8a4527c691ac.png)

**Comment by FilipePS** [comment]
Looks like a good place for this button. How did I not think of this place before?
